src/components/Modals/ConfirmModal.tsx

```diff
@@ -36,7 +36,7 @@ export const ConfirmModal: FunctionComponent<PropsWithChildren<ConfirmModalProps
   const cancelButtonRef = useRef(null);
 
   const handleOnSubmit = () => {
-    onSubmit && onSubmit();
+    onSubmit?.onSubmit();
```

Comment:
Actually this solution was given in SoundCloud , please have a look
https://sonarcloud.io/project/issues?resolved=false&id=asyncapi_studio&open=AYhPsyHDlY3YZKcDC9_D&tab=how_to_fix

Reply:
Yeah, but it's not the same :) You can use the `?.` operator to access a property of an object that may be undefined. You don't have such an object here. It's just calling a function, the `onSubmit()` function. It could be something like `onSubmit?.()` but that doesn't work.

Reply:
I think you can just leave it like this:

```suggestion
    onSubmit();
```

Because the default value of `onSubmit` is `() => void` so it's always going to be a function.

Reply:
Thanks a lot , I was too wondering why the build gets failed due to this. Will revert it back .